src/config/networks/xdai.ts

```diff
@@ -13,10 +13,10 @@ const baseConfig: EnvironmentSettings = {
   clientGatewayUrl: 'https://safe-client.gnosis.io/v1',
   txServiceUrl: 'https://safe-transaction.xdai.gnosis.io/api/v1',
   gasPriceOracles: [],
-  rpcServiceUrl: 'https://dai.poa.network/',
-  safeAppsRpcServiceUrl: 'https://dai.poa.network/',
+  rpcServiceUrl: 'https://rpc.xdaichain.com/',
```

Comment:
This is the new recommended RPC, and we're using the same in the config-service.